x-pack/platform/plugins/shared/fleet/cypress/tasks/login.ts

```diff
@@ -18,7 +18,7 @@ import { request } from './common';
  * Credentials in the `kibana.dev.yml` config file will be used to authenticate
  * with Kibana when credentials are not provided via environment variables
  */
-const KIBANA_DEV_YML_PATH = '../../../config/kibana.dev.yml';
```

Comment:
Not sure this path was correct event before the relocation.
@elastic/fleet could you please review and fix if needed in a follow-up PR? TIA